src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -54,7 +56,8 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       templateName: config.UPDATE_BENEFICIAL_OWNER_TYPE_PAGE,
       ...appData,
       hasExistingBosMos,
-      hasNewlyAddedBosMos
+      hasNewlyAddedBosMos,
+      dateOfCreation: getRegistrationDate(appData.update?.date_of_creation as InputDate)
```

Comment:
You will need to pass in your feature flag here:
`feature_flag_relevant_period: config.FEATURE_FLAG_ENABLE_RELEVANT_PERIOD`